sdk/src/wallet/bindings/java/README.md

```diff
@@ -62,42 +62,58 @@ In order to use the library, you need to create a `Wallet` instance.
 **Note**: Android applications must necessarily configure a suitable storage path for the wallet to avoid problems with file system permissions. Specify a suitable storage path as illustrated below:
 
 ```java
-// Copyright 2022 IOTA Stiftung
+// Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
 import org.iota.Wallet;
+import org.iota.types.AccountAddress;
 import org.iota.types.AccountHandle;
 import org.iota.types.ClientConfig;
 import org.iota.types.CoinType;
 import org.iota.types.WalletConfig;
+import org.iota.types.exceptions.InitializeWalletException;
 import org.iota.types.exceptions.WalletException;
 import org.iota.types.secret.StrongholdSecretManager;
 
-public class CreateAccount {
-    
-    private static final String DEFAULT_DEVELOPMENT_MNEMONIC = "hidden enroll proud copper decide negative orient asset speed work dolphin atom unhappy game cannon scheme glow kid ring core name still twist actor";
+public class GettingStarted {
+    // A name to associate with the created account.
+    private static final String ACCOUNT_ALIAS = "Alice";
 
-    public static void main(String[] args) throws WalletException, InitializeWalletException {
-        // Set a suitable storage path for the wallet to avoid problems with file system permissions.
-        // Android applications must necessarily configure this: make sure you replace the ´com.example.myapplication´ with your own app naming.
-        String storagePath = "/data/data/com.example.myapplication/";
+    // The node to connect to.
+    private static final String NODE_URL = "https://api.testnet.shimmer.network";
+
+    // Set a suitable storage path for the wallet to avoid problems with file system permissions.
+    // INFO: Android applications must necessarily configure this: make sure you replace the ´com.example.myapplication´ with your own app naming.
+    private static final String STORAGE_PATH = "/data/data/com.example.myapplication/";
+
+    // A password to encrypt the stored data.
+    // WARNING: Never hardcode passwords in production code.
+    private static final String STRONGHOLD_PASSWORD = "a-secure-password";
 
+    // The path to store the account snapshot.
+    private static final String STRONGHOLD_SNAPSHOT_PATH = "stronghold/vault.stronghold";
```

Comment:
we should remove the stronghold/ prefix,as the other examples dont have it either

Reply:
Removed